work-scheduler-data/src/main/java/com/github/gaalgergo/workscheduler/data/entities/EmployeeEntity.java

```diff
@@ -3,13 +3,22 @@
 import lombok.Data;
 import lombok.EqualsAndHashCode;
 
+
 import javax.persistence.Entity;
 import javax.persistence.Table;
+import java.time.LocalDate;
+
 
 @Data
 @EqualsAndHashCode(callSuper = true)
 @Entity
 @Table(name = "employees")
 public class EmployeeEntity extends BaseEntity {
+
     private String name;
+    private String email;
+    private LocalDate birthday;
+    private int personalID;
```

Comment:
Hungarian personal identity identifiers can contain letters and numbers.
We should change the type of this to `String`.